README.md

```diff
@@ -0,0 +1,5 @@
+## Introduction
+App info extractor is a utility to extract and analyze customer reviews and critical metrics of apps from multiple sources like google play store, Apple play store, and any other sources.This utility is highly configurable to extract and give relevant information about any app in a meaningful way.
+
+## How it works?
+This utility takes config.yaml as input. Configuration for each app is self-contained in configs. Once a configuration is specified in config.yaml, the utility will iteratively go through each app, crawl the relevant information, and store the data locally as a CSV file.
```

Comment:
Would be  helpful to give sample usage CLI 
`>command name agr1 arg2`

Reply:
done